### DISCUSSION TITLE
frontend framework

### DISCUSSION THREAD

**Discussion by erikgalajda** [Nerds-Who-Code/Mental-Health-Tracker#5] [Polls]
We should decided whether we use a frontend framework (react, vue) or write pure HTML/CSS/JS separately. I would go with react but it might be tricky if majority of us does not have experience with that.

**Comment by xintamosaik** [comment]
Oh no, right now 3 options each have 1 vote.. XD

**Reply by xintamosaik** [reply]
For now I can only use vanilla js. In a couple of weeks I might start learning react, but I am not sure if that will work out.

**Reply by 1Vera1** [reply]
Who has experience in react and/or vue? Vue is supposedly easier to learn...

**Reply by steph-koopmanschap** [reply]
I got basic understanding of both React and Vue. I can confirm Vue is easier to learn. In general React is more popular in the USA while Vue is more popular internationally. Both are nearly the same tho.
